libmamba/include/mamba/core/solver.hpp

```diff
@@ -95,6 +82,7 @@ namespace mamba
         Solver* m_solver;
         Pool* m_pool;
         Queue m_jobs;
+        MProblemsGraphs m_problems_solver;
```

Comment:
I think it would make sense to add it as a member (not sure if in this class though). if we want to query/get more info from the graphs - eg. extra verbosity of things we tried, the full dependency chain of packages etc. I can remove it from here though.